frontend/components/ArticleBody.tsx

```diff
@@ -284,11 +326,26 @@ const ArticleBody: React.SFC<{
                 />
             </div>
             <div className={meta}>
-                <div className={profile(pillarColour)}>{CAPI.author}</div>
-                <div className={twitterHandle}>
-                    {/* TODO - from the contributor type tag */}
-                    <TwitterIcon /> @ByRobDavies
+                <div className={profile(pillarColour)}>
+                    <span
+                        className={byline}
+                        dangerouslySetInnerHTML={{
```

Comment:
I wouldn't use `dangerouslySetInnerHTML` with `renderByline` returning a string here, as this mixing markup in a string template with JSX markup. You could replace `renderByline` with a function that returns a stateless React component. If you call this function `Byline` for example you can then put `<Byline byline={byline} tags={CAPI.tags} />` as a child of this span (not you can pass `byline` and `tags` as props.

Reply:
👆 

@philmcmahon I'd be happy to pair with you on this today if you like

Reply:
Gosh, this is more complicated than it looks. Id say don't worry about this for now, it can be an interesting coding challenge for us one day!

Reply:
I think there's a good case for turning this into an array of structured data and then mapping that. 
`
[ 'Firstname Lastname', 
  'and',
  {?:'Other Name',url:'https://'}
]`